index.js

```diff
@@ -46,6 +53,8 @@ async function restartable (factory, opts, fastify = defaultFastify) {
     await closeApplication(app)
 
     app = newApp
+
+    onRestartHooks.forEach(hook => hook(app, restartOptions))
```

Comment:
why is this sync?

Reply:
I rewrote this code, but the idea is that the `restart` function shouldn't wait for `onRestartHooks` execution. Does it make sense?